LICENSE.txt

```diff
@@ -9,7 +9,7 @@ software.
 
 1. Definitions
 
-The terms "reproduce," "reproduction," "derivative works," and "distribution"
+The terms "reproduce", "reproduction", "derivative works" and "distribution"
```

Comment:
Are we permitted to change that, even such a small change, and still claim
it is Ms-PL?